includes/classes/usu.php

```diff
@@ -592,30 +593,11 @@ protected function get_product_name($pID, $cID = null)
                 break;
 
             default:
-                if (USU_FORMAT == 'parent') {
-                    $sql =
-                        "SELECT pd.products_name AS pName, ptc.categories_id AS c_id, p.master_categories_id AS master_id
-                           FROM " . TABLE_PRODUCTS_DESCRIPTION . " pd
-                                LEFT JOIN " . TABLE_PRODUCTS . " p
-                                    ON p.products_id = pd.products_id
-                                LEFT JOIN " . TABLE_PRODUCTS_TO_CATEGORIES . " ptc
-                                    ON ptc.products_id = pd.products_id
-                          WHERE pd.products_id = $pID
-                            AND pd.language_id = {$this->languages_id}" . $categories_clause . "
```

Comment:
You 'lost' the `$categories_clause` in the 'parent' type formatting.  What USU_FORMAT are you using when you test?

Reply:
ok, mini rant! i HATE it when functions are only available on the store side and not on the admin side, or vice-versa or when they are different!  rant-over.

i have tested both original and parent.  see next comment.

Reply:
see below.  i had a mini-rant over functions available on store v admin side and not available and/or different that seems to be lost here.  in any event, we need to add the function `zen_product_in_category` to the admin side.  i'll add something extra.

in seems that, in this situation, the problem is currently resolved, in v158:
```
includes/init_includes/init_general_funcs.php:24:require DIR_WS_FUNCTIONS . 'functions_categories.php';
includes/functions/functions_categories.php:3: * functions_categories.php
admin/includes/init_includes/init_general_funcs.php:30:require DIR_FS_CATALOG . DIR_WS_FUNCTIONS . 'functions_categories.php';
```

Reply:
tested on both original and parent.

Reply:
> 
> 
> ok, mini rant! i HATE it when functions are only available on the store side and not on the admin side, or vice-versa or when they are different! rant-over.
> 
> i have tested both original and parent. see next comment.

It's (of course) much, much worse when the same function provides different functionality storefront vs. admin!  I'll finish my review of your changes later today or in the morning.